user-interface/src/data-verification/ConsolidationOrderAccordion.test.tsx

```diff
@@ -264,7 +286,17 @@ describe('ConsolidationOrderAccordion tests', () => {
     );
     fireEvent.click(adminRadioButton);
 
-    const modalApproveButton = screen.getByTestId('toggle-modal-button-submit');
+    const modalApproveButton = screen.getByTestId(
+      `button-confirmation-modal-${order.id}-submit-button`,
+    );
+
+    await waitFor(() => {
+      expect(modalApproveButton).toBeEnabled();
+    });
+
+    // Click button on First screen of Modal
+    fireEvent.click(modalApproveButton);
+    // Click button on Second screen of Modal
```

Comment:
```suggestion
    // click "continue" button on first screen of the modal.
    fireEvent.click(modalApproveButton);
    // click "verify" button on second screen of the modal.
```

The comments above seem more expressive.